src/selectors/getChildren.ts

```diff
@@ -74,3 +80,38 @@ export const getChildrenRanked = (state: State, context: Context): Child[] =>
 /** Returns the first visible child of a context. */
 export const firstVisibleChild = (state: State, context: Context) =>
   getChildrenSorted(state, context)[0]
+
+/** Checks if a child lies within the cursor path. */
+const isChildInCursor = (state: State, path: Path, simplePath: SimplePath, child: Child | ThoughtContext) => {
+  const showContexts = isContextViewActive(state, pathToContext(path))
+  const childSimplePath = getChildPath(state, child, simplePath, showContexts)
+  const childPath = appendChildPath(state, childSimplePath, path)
+  return state.cursor && equalThoughtRanked(state.cursor[childPath.length - 1], head(childPath))
+}
+
+/** Returns head of context if parent has active context view. */
+const getResolvedValue = (state: State, path: Path, child: Child | ThoughtContext) => {
```

Comment:
Since this wants a `Context` rather than a `Path`, it would be better to accept a `Context` directly rather than accept a `Path` and then convert it to a `Context`.

Reply:
Agreed!